content/en/docs/contribute/style/style-guide.md

```diff
@@ -108,6 +108,15 @@ events are recorded with an associated "stage". | events are recorded with an as
 The copy is called a "fork". | The copy is called a "fork."
 {{< /table >}}
 
+### Use title case for page titles {#title-case-titles}
+
+{{< table caption = "Do and Don't - Use title case for page titles" >}}
+Do | Don't
+:--| :-----
+Kubernetes API Server Bypass Risks | Kubernetes API server bypass risks
+Configure a Security Context for a Pod or Container | Configure a security context for a Pod or container
+{{< /table >}}
+
```

Comment:
Can we clarify that this is just for the title in the front matter, and not for headings within the page?

Reply:
Should this convention apply to all titles, page and heading? I dont think that we have a guideline for headings (i couldn't see it in the style guide, please point it out to me if its there already) but it seems to be the general convention that we use in headings as well

Reply:
We [write headings (not titles) in sentence case](https://kubernetes.io/docs/contribute/style/style-guide/#headings).

Reply:
The guidance on page titles could link to the advice on headings, maybe also vice versa?

Reply:
Might be a good idea to merge the titles section into the heading section, call it "Headings and titles" maybe, and clarify what works where. 

And I'd still like us to discuss changing the guidance for titles to use sentence case as well, but perhaps as a separate issue. The 5 votes that got in Slack arent many though